documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,87 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers
+container. Currently `systemd` is the default init process in the Clear Containers mini-os
+system. The Systemd project provides a tool called `systemd-bootchart` that
```

Comment:
@amshinde I was pretty sure I've seen a comment where someone tell me about change it,  but I cant see it . Changed to 'systemd'  :P